src/imputation/imputation_main.py

```diff
@@ -51,12 +39,12 @@ def run_imputation(
     df["imp_marker"] = "no_imputation"
 
     # Create new columns to hold the imputed values
-    orig_cols = target_vars + bd_cols + sum_cols
+    orig_cols = lf_target_vars + bd_cols + sum_cols
     for col in orig_cols:
         df[f"{col}_imputed"] = df[col]
```

Comment:
(I realise this might not be your code)

Suggest replacing with 

```
# Put values of target variables in the columns for imputed values
df[[f"{col}_imputed" for col in orig_cols]] = df[orig_cols]
```